installing/installing_aws/installing-aws-localzone.adoc

```diff
@@ -0,0 +1,228 @@
+.Additional resources
+
+* You can view details about the CloudFormation stacks that you create by navigating to the link:https://console.aws.amazon.com/cloudformation/[AWS CloudFormation console].
+
+* You can view details about your hosted zones by navigating to the link:https://console.aws.amazon.com/route53/[AWS Route 53 console].
+
+* See link:https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/ListInfoOnHostedZone.html[Listing public hosted zones] in the AWS documentation for more information about listing public hosted zones.
+
+include::modules/installation-creating-aws-security.adoc[leveloffset=+1]
+
+include::modules/installation-cloudformation-security.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* You can view details about the CloudFormation stacks that you create by navigating to the link:https://console.aws.amazon.com/cloudformation/[AWS CloudFormation console].
+
+include::modules/installation-aws-ami-stream-metadata.adoc[leveloffset=+1]
+
+include::modules/installation-aws-user-infra-rhcos-ami.adoc[leveloffset=+1]
+
+include::modules/installation-aws-regions-with-no-ami.adoc[leveloffset=+2]
+
+include::modules/installation-aws-upload-custom-rhcos-ami.adoc[leveloffset=+2]
+
+include::modules/installation-creating-aws-bootstrap.adoc[leveloffset=+1]
+
+include::modules/installation-cloudformation-bootstrap.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* You can view details about the CloudFormation stacks that you create by navigating to the link:https://console.aws.amazon.com/cloudformation/[AWS CloudFormation console].
+
+* See xref:../../installing/installing_aws/installing-aws-user-infra.adoc#installation-aws-user-infra-rhcos-ami_installing-aws-user-infra[{op-system} AMIs for the AWS infrastructure] for details about the {op-system-first} AMIs for the AWS zones.
+
+include::modules/installation-creating-aws-control-plane.adoc[leveloffset=+1]
+
+include::modules/installation-cloudformation-control-plane.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* You can view details about the CloudFormation stacks that you create by navigating to the link:https://console.aws.amazon.com/cloudformation/[AWS CloudFormation console].
+
+include::modules/installation-creating-aws-worker.adoc[leveloffset=+1]
+
+////
+[id="installing-workers-aws-user-infra"]
+== Creating worker nodes
+
+You can either manually create worker nodes or use a MachineSet to create worker
+nodes after the cluster deploys. If you use a MachineSet to create and maintain
+the workers, you can allow the cluster to manage them. This allows you to easily
+scale, manage, and upgrade your workers.
+////
+
+include::modules/installation-cloudformation-worker.adoc[leveloffset=+2]
+//Do we need to remove the instructions to create separate worker nodes? It looks like the installation program might make the workers because we're creating a manifest for them.
```

Comment:
I am very curious about this.

Reply:
Yes, we need to remove it. I left a comment with suggested changes to remove those items.

Reply:
I have removed this include: plus the comment.